drakrun/drakrun/draksetup.py

```diff
@@ -83,6 +83,54 @@ def check_root():
         return True
 
 
+@click.command(help='Cleanup the changes made by draksetup')
+def cleanup():
+    if not check_root():
+        return
+
+    install_info = InstallInfo.try_load()
+
+    if install_info is None:
+        logging.error("The cleanup has been performed")
+        return
+
+    backend = get_storage_backend(install_info)
+    vm_ids = get_all_vm_conf()
+
+    for vm_id in vm_ids:
+        delete_vm_conf(vm_id)
+        vm = VirtualMachine(backend, vm_id)
+        vm.destroy()
+
+        net_enable = int(conf['drakrun'].get('net_enable', '0'))
+        out_interface = conf['drakrun'].get('out_interface', '')
+        dns_server = conf['drakrun'].get('dns_server', '')
+
+        delete_vm_network(vm_id=vm_id, net_enable=net_enable, out_interface=out_interface, dns_server=dns_server)
+        if net_enable:
+            stop_dnsmasq(vm_id=vm_id)
```

Comment:
This code isn't going to be executed. We don't run dnsmasq in the background (since it's a bad idea)

Reply:
Oh, I had just seen the flag of `background` so made this code. The code can still stay there right? Or do I have to remove this also

Reply:
Ok, taking that back. We're starting it during setup. Then I guess this should stay